### DISCUSSION TITLE
Get-OvLogicalInterconnectGroup -Name "XXXXXXXXX" -ErrorAction Stop | Set-OVUplinkSet -UplinkSetName "YYYYYYYYYY" -AddNetwork "ZZZZZZZZZZZZZ TaskState Error"

### DISCUSSION THREAD

**Discussion by kymra27** [HewlettPackard/POSH-HPOneView#620] [Q&A] [answered]
Hello,

I contact you for this problem. when I add a network via the command:

Get-OvLogicalInterconnectGroup -Name "XXXXXXXXX" -ErrorAction Stop | Set-OVUplinkSet -UplinkSetName "YYYYYYYYYY" -AddNetwork "ZZZZZZZZZZZZZZ"

I receive in return an error message which indicates that the TaskState is in error.
I don't have any details to solve this problem.

I logged into GUI and in Activity my error does not appear.

Do you have an idea ?

Regards,

![2022-11-03 17_49_02-10 3 50 40 - Connexion Bureau à distance](https://user-images.githubusercontent.com/67340494/199784505-e351f865-3f2e-49da-b531-47a87f6120b9.png)

**Comment by ChrisLynchHPE** [comment, marked as answer]
You need to capture the task object into a variable, then look at the `.taskErrors` property:

```powershell
$task = Get-OvLogicalInterconnectGroup -Name "XXXXXXXXX" -ErrorAction Stop | Set-OVUplinkSet -UplinkSetName "YYYYYYYYYY" -AddNetwork "ZZZZZZZZZZZZZZ"
 if ($task.taskState -ne "Complete") {

    $task.taskErrors | ForEach { 
        Write-Host "Error Message" $_.details
        Write-Host "Recommended actions:" $_.recommendedActions
    }

}
```

**Reply by kymra27** [reply]
Wow , thank you for your reactivity !!!

I resolve my problem. Another administrator had already created a network with the same VLAN ID that's why I got an error.

Thanks a lot for your help 👍

**Reply by ChrisLynchHPE** [reply]
You're welcome. Glad to have this resolved.
